### PR TITLE
Feature/add template documentation

### PR DESCRIPTION
**Pull request by rudionrails**
Scaffold responses, documentation & specs

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,10 +2,21 @@ source 'https://rubygems.org/'
 
 ruby File.read('.ruby-version').match(/\d\.\d.\d/)[0]
 
-# App Stack
+# pplication
```
Comment (cs) [inline Gemfile]: spelling

File: spec/requests/v1/get_oid_customers_id_spec.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+RSpec.describe 'GET /v1/:oid/documents/:id', type: :request do
```
Comment (cs) [inline spec/requests/v1/get_oid_customers_id_spec.rb]: File name doesn't match with the content.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -1,19 +1,12 @@
...
-%w(./config/initializers ./lib).each do |load_path|
-  Find.find(load_path) { |f| require f if f.match(/\.rb$/) }
-end
+require_relative '../config/boot'
 
 # Load support files
 Dir[File.dirname(__FILE__) + '/support/*.rb'].each { |f| require f }
 
 RSpec.configure do |config|
+  config.include JsonSpec::Helpers
```
Comment (cs) [inline spec/spec_helper.rb]: In the recurrence service I have a spec group (i.e. `type: :request`) to include stuff like this only in request specs.

**Comment by cs** [comment]
Rebased on master :)

File: spec/support/request_specs.rb [hunk 4]
```diff
@@ -1,11 +1,26 @@
+require 'rack/test'
+require 'json_spec'
```
Comment (rudionrails) [inline spec/support/request_specs.rb]: Just as a question: is it necessary to explicitly require json_spec again? It should already be loaded from the test group in the Gemfile

Comment (cs) [inline spec/support/request_specs.rb]: You are right. It works without the `require` statements. I've just reverted my commit. Good to merge?

File: spec/support/request_specs.rb [hunk 5]
```diff
@@ -1,11 +1,24 @@
 require_relative(File.expand_path('../../../app/service', __FILE__))
```
Comment (cs) [inline spec/support/request_specs.rb]: One more thing: We don't need `require_relative` AND `File.expand_path`, right? That's redundant?

Comment (rudionrails) [inline spec/support/request_specs.rb]: did you already change this? I would also only use `require_relative '../file'`. I can merge it into master then :-)

Comment (cs) [inline spec/support/request_specs.rb]: No, I did not.

**Comment by cs** [comment]
Rebased on master :)

**Pull request merged**